server/chat.ts

```diff
@@ -1914,4 +1946,5 @@ export const Chat = {
 	stripFormatting,
 
 	updateServerLock,
+	version,
```

Comment:
Can we move these to ``Monitor``?

Reply:
https://github.com/Zarel/Pokemon-Showdown/search?q=updateServerLock&unscoped_q=updateServerLock

Reply:
Done.